build.gradle

```diff
@@ -484,6 +484,7 @@ tasks.register('testIntegration', Test) {
     }
     maxParallelForks = 8
     outputs.upToDateWhen { false }
+    systemProperty 'junit.jupiter.execution.parallel.enabled', 'true'
```

Comment:
This is required to enable concurrent test execution. If we decide to use concurrent execution for other test types (like connected tests) this configuration can be moved to a shared scope, or moved to a property file.

Reply:
Is it the case that this just allows for parallel execution and tests still need to explicitly make use of it?